### PR TITLE
Adds a new jacket for the Commanding Officer

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Adds a jacket for CO

Requested by Dargus / Superreallycoolguy
# Explain why it's good for the game

More clothes and customization

# Changelog
:cl:
add: Added a new CO jacket to the CO vendor
/:cl:

### CODE REVIEW

**Comment by c4xmaniac2** [comment]
Wholesome speed merge now

File: code/modules/gear_presets/uscm_ship.dm [hunk 1]
```diff
@@ -476,6 +476,8 @@
 	role_comm_title = "CO"
 	minimum_age = 35
 
+	dress_over = list(/obj/item/clothing/suit/storage/jacket/marine/dress/officer/white, /obj/item/clothing/suit/storage/jacket/marine/dress/officer/black, /obj/item/clothing/suit/storage/jacket/marine/dress/officer/suit, /obj/item/clothing/suit/storage/jacket/marine/dress/officer/falcon)
```
Comment (harryob) [inline code/modules/gear_presets/uscm_ship.dm:479]: structure this as a long list
```
list(
    entry,
    entry,
)
```

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/gear_presets/uscm_ship.dm [hunk 2]
```diff
@@ -476,6 +476,13 @@
 	role_comm_title = "CO"
 	minimum_age = 35
 
+	dress_over = list(
+		/obj/item/clothing/suit/storage/jacket/marine/dress/officer/white,
+		/obj/item/clothing/suit/storage/jacket/marine/dress/officer/black,
+		/obj/item/clothing/suit/storage/jacket/marine/dress/officer/suit,
+		/obj/item/clothing/suit/storage/jacket/marine/dress/officer/falcon
```
Comment (BeagleGaming1) [inline code/modules/gear_presets/uscm_ship.dm:483]:
```suggestion
		/obj/item/clothing/suit/storage/jacket/marine/dress/officer/falcon,
```

File: code/modules/gear_presets/uscm_ship.dm [hunk 3]
```diff
@@ -476,6 +476,13 @@
 	role_comm_title = "CO"
 	minimum_age = 35
 
+	dress_over = list(
```
Comment (morrowwolf) [inline code/modules/gear_presets/uscm_ship.dm:479]: Probably don't want this under dress

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/gear_presets/uscm_ship.dm (hunk 3 above)
Comment (morrowwolf) [reply code/modules/gear_presets/uscm_ship.dm:479]: Actually this entire vendor is really weirdly organized, whatever

**Pull request merged**